.semaphore/semaphore.yml

```diff
@@ -21,6 +21,9 @@ blocks:
       jobs:
         - name: Build & Test
           commands:
+            # configure file limits
+            - ulimit -S -n 1024000
+            - echo "fs.file-max = 1024000" | sudo tee -a /etc/sysctl.conf
```

Comment:
Now that we have fixed the test issues in #1400 I wonder if this is really necessary. If it is, why do the test pass? Shouldn't the "too many open files" error completely break the tests?

Reply:
> Shouldn't the "too many open files" error completely break the tests?

It looks like it does, but it makes our test suite hang so we don't cleanly witness it (because we kill the build after 15 minutes to avoid wasting resources).

> I wonder if this is really necessary

I can't be entirely sure, but since I added that I haven't repro-ed the hang on semaphore (I restarted the test suite several times), so it looks like it happens less often (hopefully it doesn't happen anymore). At least it can't hurt to raise this limit (the ulimit is already at 1024000 by default in semaphore, but it wasn't configured in sysctl.conf).

Reply:
Ok, so this fixes the intermittent hanging while #1400 introduced then fixed a permanent hanging.